service-worker/worker/src/testing/mock_cache.ts

```diff
@@ -230,7 +230,7 @@ export class MockResponse extends MockBody implements Response {
   url: string;
   headers: any;
   type: string = "default";
```

Comment:
This is true from a purely behavior standpoint, TS will infer the correct type.

For well-defined interfaces (like Response which is part of the fetch spec), I prefer to specify the type explicitly because it acts as an assertion that the initialization value conforms to the spec.